airbyte-integrations/connectors/source-google-ads/acceptance-test-config.yml

```diff
@@ -20,18 +20,35 @@ acceptance_tests:
       - config_path: "secrets/config.json"
         expect_records:
           path: "integration_tests/expected_records.jsonl"
-        timeout_seconds: 600
+        timeout_seconds: 3600
         empty_streams:
-          - name: "display_topics_performance_report"
-            bypass_reason: "Stream not filled yet."
           - name: "account_labels"
-            bypass_reason: "Unable to seed the stream"
+            bypass_reason: "Unsufficient permissions"
```

Comment:
Please include the specific permissions we're lacking as the reason for bypassing the testing of this stream.